src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/NodeFileTransferTests.cs

```diff
@@ -21,153 +21,174 @@
-                new TransferFileBytesRequestObserver(_nodeFileTransferFactory, peerSettings, _logger);
-
-            var uniqueFileKey = CorrelationId.GenerateCorrelationId();
-            var crcValue = FileHelper.GetCrcValue(fileToTransfer);
-
-            //Create a response object and set its return value
-            var request = new AddFileToDfsRequest
-            {
-                Node = "node1",
-                FileName = fileToTransfer,
-                FileSize = (ulong) byteSize
-            }.ToProtocolMessage(sender, uniqueFileKey);
-            request.SendToHandler(_fakeContext, addFileToDfsRequestHandler);
-
-            Assert.Single(_nodeFileTransferFactory.Keys);
-
-            var fileTransferInformation =
-                _nodeFileTransferFactory.GetFileTransferInformation(uniqueFileKey);
-            Assert.True(fileTransferInformation.Initialised, "File transfer not initialised");
-
-            using (var fs = File.Open(fileToTransfer, FileMode.Open))
-            {
-                var fileUploadInformation = new UploadFileTransferInformation(fs, senderPeerId, recipientPeerId,
-                    fakeNode.Channel, uniqueFileKey);
-                for (uint i = 0; i < fileTransferInformation.MaxChunk; i++)
-                {
-                    fileUploadInformation.GetUploadMessageDto(i).Content
-                       .SendToHandler(_fakeContext, transferBytesRequestHandler);
-                }
-            }
-
-            Assert.True(fileTransferInformation.ChunkIndicatorsTrue());
-
-            await TaskHelper.WaitForAsync(() => fileTransferInformation.DfsHash != null, TimeSpan.FromSeconds(15)).ConfigureAwait(false);
-            Assert.NotNull(fileTransferInformation.DfsHash);
-
-            long ipfsCrcValue;
-            using (var ipfsStream = await _dfs.ReadAsync(fileTransferInformation.DfsHash).ConfigureAwait(false))
-            {
-                ipfsCrcValue = FileHelper.GetCrcValue(ipfsStream);
-            }
-
-            Assert.Equal(crcValue, ipfsCrcValue);
-        }
-    }
-}
+// #region LICENSE
+//
+// /**
+// * Copyright (c) 2019 Catalyst Network
+// *
+// * This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+// *
+// * Catalyst.Node is free software: you can redistribute it and/or modify
+// * it under the terms of the GNU General Public License as published by
+// * the Free Software Foundation, either version 2 of the License, or
+// * (at your option) any later version.
+// *
+// * Catalyst.Node is distributed in the hope that it will be useful,
+// * but WITHOUT ANY WARRANTY; without even the implied warranty of
+// * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// * GNU General Public License for more details.
+// *
+// * You should have received a copy of the GNU General Public License
+// * along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+// */
+//
+// #endregion
+//
+// using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)